src/spec/MaterialRendererSpec.vitest.ts

```diff
@@ -439,7 +436,10 @@ describe('A Material', () => {
 
     expect(material).toBeDefined();
     expect(logger.warn).toHaveBeenCalledWith(
-      'Max number texture slots 31 have been reached for material "test", no more textures will be uploaded due to hardware constraints.'
+      expect.stringMatching(
+        // this was 31 before, but since moving to vite it's 15. Is there a difference in max texture slots from chromium vs old karma browser?
+        /Max number texture slots (\d+) have been reached for material "test", no more textures will be uploaded due to hardware constraints\./
+      )
```

Comment:
@eonarheim i'm unsure why max texture slots would have changed from 31 to 15 from karma chrome -> playwright chromium

Reply:
It might be down to chrome's configuration to chromiums? It also could be based on the driver/hardware on your system